### ISSUE TITLE
Update zend-component-installer version to 0.7 in composer.json

### ISSUE THREAD

**Issue by seyfer** [zendframework/ZendSkeletonApplication#411]
Please, update 

`"zendframework/zend-component-installer": "^1.0 || ^0.3 || ^1.0.0-dev@dev",`

to 

`"zendframework/zend-component-installer": "^1.0 || ^0.5 || ^1.0.0-dev@dev",`

in order to solve this issue

https://github.com/zendframework/zend-component-installer/issues/28

thanks!

**Comment by seyfer** [comment]
It's really important in order to fix this warning message

```
Deprecation Notice: The callback Zend\ComponentInstaller\ComponentInstaller::onPostPackageInstall declared at /home/runner/cdnperf.com/vendor/zendframework/zend-component-installer/src/ComponentInstaller.php 
```

which outputs on every component install in version 0.3

![screenshot from 2017-03-09 17-42-17](https://cloud.githubusercontent.com/assets/1483925/23746810/e68e37ee-04ef-11e7-94ba-9a9f761ab242.png)

**Comment by Xerkus** [comment]
bump it to `^0.6` instead

**Comment by webimpress** [comment]
I'd even suggest the latest so `^0.7`.

**Comment by seyfer** [comment]
I have tested skeleton application install with `^0.7` and it work's like a charm.

According to changelog
https://github.com/zendframework/zend-component-installer/blob/0.7.0/CHANGELOG.md

there are no critical changes, only fixes. so, it could be safely bumped to `0.7`

**Comment by seyfer** [comment]
We have a problem with zend-skeleton-installer

```
zendframework/zend-skeleton-installer 0.1.3 requires zendframework/zend-component-installer ^1.0 || ^0.3 || ^1.0.0-dev@dev -> satisfiable by zendframework/zend-component-installer[1.0.x-dev].
```

it requires version 0.3 of zend-component-installer. so, it's dependency should be bumped to `0.7` also.

**Comment by seyfer** [comment]
I have added the same issue and pull request into zend-skeleton-installer repository.

https://github.com/zendframework/zend-skeleton-installer/issues/8

**Comment by webimpress** [comment]
I think this is duplicate of #403

**Comment by seyfer** [comment]
@webimpress well, I have considered version bump for zend-skeleton-installer also and added connected pull request. This is the difference.

**Comment by Xerkus** [comment]
Fixed in #412
